edit-post/components/sidebar/post-status/index.js

```diff
@@ -32,6 +33,9 @@ function PostStatus( { isOpened, onTogglePanel } ) {
 			<PostSticky />
 			<PostPendingStatus />
 			<PostAuthor />
+			<PanelRow>
```

Comment:
I don't think this should be included here. At the moment every item in this Panel is wrapped with `PanelRow`. Instead, we can expose `Fill` which is wrapped with `PanelRow`. In effect, you would add new items as follows:

```js
<PluginPostStatusVisibility>
    My item
< PluginPostStatusVisibility>
```

and `Fill` would work as follows:
```js 
const PluginPostStatusVisibility = ( { children } ) =>  (
    <Fill>
        <PanelRow>
            { children }
        </PanelRow>
    </Fill>
);
```

Alternative solution would be to leave it to plugin developers in case they wanted to add more than one item:

```js
<PluginPostStatusVisibility>
    <PanelRow>My item 1</PanelRow>
    <PanelRow>My item 2</PanelRow>
< PluginPostStatusVisibility>
```

Reply:
@gziolo I was thinking the same thing, I would prefer to have a single component. I am new using createSlotFill and am not sure how to implement what you are asking for. Where does 
```
const PluginPostStatusVisibility = ( { children } ) =>  (
    <Fill>
        <PanelRow>
            { children }
        </PanelRow>
    </Fill>
);
```
go?

Reply:
It can go in here: https://github.com/WordPress/gutenberg/pull/6300/files#diff-ee98bfe279b9987ab8753b1c2fd680f0R7

You will have to juggle a bit with exports.

Reply:
@gziolo can you provide some examples of how I would be able to do this - specifically the reference to juggling the exports?